lib/redix/pubsub/connection.ex

```diff
@@ -162,6 +162,18 @@ defmodule Redix.PubSub.Connection do
     end
   end
 
+
+  # Ask Redix.Utils to setup a connection, then configure the returned
+  # socket with http://erlang.org/doc/man/inet.html#setopts-2
+  #
+  @spec establish_connection(Keyword.t) :: {:ok, :gen_tcp.socket} | {:error, term} | {:stop, term}
```

Comment:
Let's see what @whatyouhide will say. I'd personally remove it. Without Dialyzer usage specs for private function are less useful, and tend to go out of sync with actual return.